client/src/featureform/sqlite_metadata.py

```diff
@@ -193,7 +193,7 @@ def query_resource(self, type, column, resource):
         variant_data = self.__conn.execute(query)
         self.__conn.commit()
         variant_data_list = variant_data.fetchall()
-        if len(variant_data_list) == 0:
+        if len(variant_data_list) == 0 and column != "owner":
```

Comment:
Shouldnt there be the default user though?

Reply:
This checks the owner column in resource metadata tables. For example if a user is registered but doesn't create any resources the query_resource will return an empty list. So in the feature_variant table there are chances there is no feature owned by ____.